app/modules/ads/tracking/slot-tracker.js

```diff
@@ -10,8 +10,6 @@ const onChangeStatusToTrack = [
   'catlapsed',
   'error',
   'viewport-conflict',
```

Comment:
`'viewport-conflict'` has also been removed from status.
https://github.com/Wikia/ad-engine/pull/221/files#diff-3755f3ded23b967a2b9a6ae83dee2bb8L124

Reply:
I'm not really sure if it needs a change.